src/utils/jsonHandler.ts

```diff
@@ -55,4 +70,59 @@ export async function getConfig(filename: string, callback: (config: Configurati
     } else {
         callback(undefined) // file not found
     }
+}
+
+/**
+ * Method to open/create and modify a json file containing thread information
+ * 
+ * @param filename name of the thread file
+ * @param thread the thread with all of the interactions
+ * @param message message contents and from who
+ */
+export function openThreadInfo(filename: string, thread: ThreadChannel, messages: UserMessage[] = []) {
+    // check if the file exists, if not then make the config file
+    const fullFileName = `data/${filename}`
+    if (fs.existsSync(fullFileName)) {
```

Comment:
I think most of this use of fs, especially checking for existing, will greatly benefit from use of future redis caching.